Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -237,74 +238,20 @@ public String reportProblemForAll() {
         for (Task s : this.steps) {
             this.currentStep = s;
             this.myDav.uploadFromHome(this.currentStep.getProcess());
-            reportProblem();
+            serviceManager.getWorkflowService().setProblem(getProblem());
+            try {
+                setCurrentStep(serviceManager.getWorkflowService().reportProblem(this.currentStep, this.problemTask));
+            } catch (DataException e) {
+                logger.error("Problem couldn't be reported: " + e);
```

Comment:
2 things:
1) If a `DataException` is thrown is it a good idea to go on?
2) Did this kind of error logging work with casting exception object to string message?

Reply:
Look previous.